lighthouse-core/gather/connections/cri.js

```diff
@@ -74,10 +75,16 @@ class CriConnection extends Connection {
         });
         response.on('end', _ => {
           if (response.statusCode === 200) {
-            resolve(JSON.parse(data));
-            return;
+            try {
+              resolve(JSON.parse(data));
+              return;
+            } catch (e) {
+              if (data === 'Target is closing') {
```

Comment:
Will you add some variant of this as a comment in the code? At least for the string being checked against